lib/lighthouse/veterans_health/client.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
+
+require 'lighthouse/veterans_health/configuration'
+require 'lighthouse/veterans_health/jwt_wrapper'
+
+# This client was written to work for the specific use case of the
+# VA OCTO's hypertension fast track pilot, which is located in a Sidekiq job that is kicked off
+# one-to-one for each veteran 526 claim for increase submitted in va.gov
+# If you're looking for a more generic Veterans Health Lighthouse client that doesn't instantiate
+# with a set ICN, consider creating/using another client
+module Lighthouse
+  module VeteransHealth
+    # Documentation located at:
+    # https://developer.va.gov/explore/health/docs/fhir?version=current
+    class Client < Common::Client::Base
+      include Common::Client::Concerns::Monitoring
+      configuration Lighthouse::VeteransHealth::Configuration
+
+      # Initializes the Veterans Health client.
+      #
+      # @example
+      #
+      # Lighthouse::VeteransHealth::Client.new(12345)
+      #
+      # @param [String] icn The ICN of the veteran filing the 526 claim for increase
+      #
+      # @return [Lighthouse::VeteransHealth::Client]
+      def initialize(icn)
+        @icn = icn
+      end
+
+      # Handles the Lighthouse request for the passed-in resource.
+      #
+      # @example
+      #
+      # get_resource('conditions')
+      #
+      # @param [String] resource The Lighthouse resource being requested
+      #
+      # @return Faraday::Env response
+      def get_resource(resource)
+        resource = resource.downcase
+        unless %w[conditions medications observations].include?(resource)
+          raise ArgumentError, 'unsupported resource type'
+        end
+
+        send("get_#{resource}")
+      end
+
+      private
+
+      def get_conditions
+        params = {
+          'patient' => @icn,
+          'clinical-status' => 'http://terminology.hl7.org/CodeSystem/condition-clinical|active'
+        }
```

Comment:
Could we use the new hash syntax here?
```suggestion
        params = {
          patient: @icn,
          'clinical-status': 'http://terminology.hl7.org/CodeSystem/condition-clinical|active'
        }
```

Reply:
fixed here: https://github.com/department-of-veterans-affairs/vets-api/pull/8558/commits/989e639c0b6996c4726f43eba865e33aab5e06b6